index.js

```diff
@@ -67,35 +66,6 @@ module.exports = {
    * @property {object} rules - Tree custom rule and additional linter configuration.
    */
   rules: {
-    'jsdoc/check-alignment': 'warn',
-    'jsdoc/check-examples': 'warn',
-    'jsdoc/check-indentation': 'off',
-    'jsdoc/check-param-names': 'warn',
-    'jsdoc/check-syntax': 'warn',
-    'jsdoc/check-tag-names': 'warn',
-    'jsdoc/check-types': 'warn',
-    'jsdoc/implements-on-classes': 'warn',
-    'jsdoc/match-description': 'warn',
-    'jsdoc/newline-after-description': 'off',
-    'jsdoc/no-types': 'off',
-    'jsdoc/no-undefined-types': 'off', // 2020-01-23: This was broken in eslint-plugin-jsdoc#8 in 2019-06, and hasn't gotten much better. Disabled, for now. Check back later.
-
-    'jsdoc/require-description-complete-sentence': 'off',
-    'jsdoc/require-description': 'warn',
-    'jsdoc/require-example': 'off',
-    'jsdoc/require-file-overview': 'off',
-    'jsdoc/require-hyphen-before-param-description': 'warn',
-    'jsdoc/require-jsdoc': 'warn',
```

Comment:
I, personally, feel that setting this rule to `off` would give you the freedom you desired, but I have not tested this theory.

Reply:
I see. So it wouldn't require us to add jsdocs, but as soon as you do, the other jsdoc rules would kick in. I'd be good with that.

Reply:
![winwinwin](https://user-images.githubusercontent.com/2700098/87458762-abd13600-c5c7-11ea-8f2e-c87af6dd53c6.gif)